pytest_mozwebqa/cloud/saucelabs.py

```diff
@@ -0,0 +1,130 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ConfigParser import ConfigParser
+import os
+import json
+
+from py.xml import html
+import requests
+from selenium import webdriver
+
+
+class SauceLabs(object):
+
+    name = 'Sauce Labs'
+
+    def __init__(self):
```

Comment:
I tried this in an earlier version of my patch. In the end I preferred to keep the terminology used by the providers (api/access key) and to keep these more or less self contained.